ngraph/frontend/paddlepaddle/include/paddlepaddle_frontend/exceptions.hpp

```diff
@@ -0,0 +1,45 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <frontend_manager/frontend_exceptions.hpp>
+#include <ngraph/node.hpp>
+
+namespace ngraph
+{
+    namespace frontend
+    {
+        namespace pdpd
+        {
+            class NodeContext;
```

Comment:
Why do we need to have forward declaration here?

Reply:
For me it is a good practice to keep in header files only this info which is required for inclusion. There is no need of full definition of NodeContext here, so not necessary to include appropriate header and forward declaration is enough.
Also it reduces compilation time.